practice-app/.gitignore

```diff
@@ -0,0 +1,16 @@
+__pycache__
+# Ignore files related to API keys
+.env
+
```

Comment:
Looks fine! 
Could you please also add IDE specific extensions/directories, 
e.g. 
- for visual studio
.vscode  
- for pycharm
.idea

Reply:
With [the last commit](https://github.com/bounswe/bounswe2022group7/pull/138/commits/4f0d9d5614610fbb8aee6202ab95fcb8ca21ebe5), I added the IDE specific directories as you suggested.